src/dfx/src/config/cache.rs

```diff
@@ -181,22 +191,25 @@ pub fn install_version(v: &str, force: bool) -> DfxResult<PathBuf> {
             file.unpack_in(temp_p.as_path())
                 .context("Failed to unpack archive asset.")?;
 
-            let full_path = temp_p.join(file.path().context("Failed to get file path.")?);
-            let mut perms = std::fs::metadata(full_path.as_path())
-                .with_context(|| {
+            #[cfg(unix)]
```

Comment:
```suggestion
            // Not necessary on Windows as these permissions are set by default.
            #[cfg(unix)]
```

Reply:
I think these go in the wrong direction re: which is base knowledge and which requires clarifying, but comments here can't hurt.